CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 ## Unreleased
 
+* Feat: Automatic breadcrumbs logging for fragment lifecycle (#1522)
```

Comment:
this is 5.1.0-beta.1

Reply:
It was marked on ur review so I moved down. 

![image](https://user-images.githubusercontent.com/1633368/123096080-b8c3ef80-d3fc-11eb-846f-848ec4616d80.png)


It's quick tp fix these things btw ^

I'll fix it now